src/py42/sdk/queries/fileevents/filters/activity_filter.py

```diff
@@ -0,0 +1,39 @@
+from py42.sdk.queries.fileevents.file_event_query import FileEventFilterStringField
+from py42.sdk.queries.query_filter import filter_attributes
+
+
+class TrustedActivity(FileEventFilterStringField):
```

Comment:
you should use `QueryFilterBooleanField` here, see `Shared` for an example. No need to add any constants.

Reply:
Changed.